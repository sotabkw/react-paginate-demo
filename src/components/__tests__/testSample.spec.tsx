import React from "react"
import renderer from "react-test-renderer"
import TestSample from "../testSample"

test("TestSample", () => {
  const component = renderer.create(<TestSample />)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
