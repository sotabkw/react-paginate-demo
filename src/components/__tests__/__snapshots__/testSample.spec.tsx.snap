// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestSample 1`] = `
<h1
  className="sc-bdvvaa btYIOi"
>
  message
</h1>
`;
